src/tools/dashboard/components/Row.tsx

```diff
@@ -258,6 +265,94 @@ const NiceStatus = ({ job, jobLink, jobUrl }: NiceStatusProps) => {
   }
 };
 
+interface JobSpecificParametersProps {
+  job: Job;
+}
+
+const taxonsWithEllipsisReveal = (taxIDs: SelectedTaxon[]) => {
+  const visibleID = taxIDs[0];
+  const hiddenIDs = taxIDs.slice(1);
+
+  return (
+    <span>
+      Selected taxonomy: {visibleID.label}
+      {hiddenIDs.length ? (
+        <EllipsisReveal>
+          {', '}
+          {hiddenIDs.map((taxon) => taxon.label).join(', ')}
+        </EllipsisReveal>
+      ) : null}
+    </span>
+  );
+};
+
+const JobSpecificParamaters = ({ job }: JobSpecificParametersProps) => {
+  const { data: idMappingFields } = useDataApi<IDMappingFormConfig>(
+    apiUrls.idMappingFields
```

Comment:
```suggestion
    job.type === JobTypes.ID_MAPPING ? apiUrls.idMappingFields : undefined
```
to avoid doing a network request when it's not needed

Reply:
I had a condition outside but using hooks inside posed a problem. it is a clever way to do it inside